modules/vpc/variables.tf

```
@@ -1,85 +1,152 @@
-variable "networks" {
+variable "subnetworks" {
   description = <<-EOF
-  Map of networks, a minimal example:
+  A map containing subnetworks configuration. Subnets can belong to different regions.
+  List of available attributes of each subnetwork entry:
+  - `subnetwork_name` : Name of the subnetwork.
```

```suggestion
  - `name` : Name of the subnetwork.
```
It's within the same scope as `name` for network used previously, so maybe we can shorter this attribute name?

---

Modified the module and all of the examples to use `name` for the subnetworks map.